pkg/translate/translateValue.go

```diff
@@ -229,19 +247,135 @@ func (t *ValueYAMLTranslator) setEnablementAndNamespacesFromValue(valueSpec map[
 	return nil
 }
 
-// translateTree is internal method for translating value.yaml tree
-func translateTree(valueTree map[string]interface{},
+// translateHPASpec translates HPA related configurations from helm values.yaml tree
+func translateHPASpec(inPath string, outPath string, value interface{}, valueTree map[string]interface{}, cpSpecTree map[string]interface{}) error {
+	asEnabled, ok := value.(bool)
+	if !ok {
+		return fmt.Errorf("expect autoscaleEnabled node type to be bool but got: %T", value)
+	}
+	if asEnabled {
+		newP := strings.Split(inPath, ".")[0]
```

Comment:
better to use util.Path since it avoids hard-coding the path separator.

Reply:
done